packages/seacas/libraries/exodus/src/ex_inquire.c

```diff
@@ -975,6 +975,18 @@ static int ex_inquire_internal(int exoid, int req_info, int64_t *ret_int, float
 #endif
     break;
 
+  case EX_INQ_FILE_FORMAT:
```

Comment:
If this stays, then it should also distinguish between all the different formats.  classic, netcdf4-classic, netcdf4, netcd3-large-model, netcdf5.

I'm not sure it is needed though, see comment below...

Reply:
I think the only change here is to remove the `#if NC_HAS_HDF5`.  The `nc_inq_format` function call exists in non-hdf5-enabled builds and will return the format which can just be returned...

Reply:
Yes, it is being being used as part of the check for whether an input Exodus database can actually support groups or not (as opposed to contains groups). Anything return value for the format other than netcdf4 is treated as false. I can remove the `#if` statement

Reply:
Yes, for this use case, but if it becomes part of the Exodus API, then it needs to work for all supported exodus formats.  Removing the #if should do that.  There are about 10 possible formats for the underlying netCDF file (most not commonly used yet).